### DISCUSSION TITLE
Access pictures/files in resource DLL

### DISCUSSION THREAD

**Discussion by andre0707** [MicrosoftEdge/WebView2Feedback#3890] [Q&A]
With the old html control, I was able to access pictures/files in my resource DLL inside a html document like this

```
<html>
<body>
<h1>Some headline</h1>
Here is the image:<br>
<img src="res://MyResource/MyPicture.png">
</body>
</html>
```

This does not work with the WebView2. What do I need to change, to make it work again?

**Comment by ukandrewc** [comment]
You'll need to extract the images and either save them to file (in which case you'll need to look up setting local protocol) or (simplest) insert them as base64.
